### ISSUE TITLE
Typescript (seems to be) missing "attachment" property.

### ISSUE THREAD

**Issue by dylan-baskind** [danreeves/react-tether#206]
Hey y'all,

I might be missing something but using TypeScript it seems that `react-tether.d.ts` is missing a type definition for the `attachment` property - i.e. I had to dig into node_modules and update the file with:

```
  interface TetherComponentProps
    extends React.Props<TetherComponent>,
      Tether.ITetherOptions {
    renderTarget?: RenderProp;
    renderElement?: RenderProp;
    renderElementTag?: string;
    attachment?: string; // I need to add this to make things work in TS
```

Might be something I'm misunderstanding though, since no one else seems to be having an issue.

**Comment by danreeves** [comment]
Yeah you might be right. I'm not really a TypeScript user so they might be a bit wrong. PRs are welcome!

**Comment by noelparisi** [comment]
I am also having this issue @dylan-baskind . I am using react-tether 2.0.1 and am using TS 2.9.x

**Comment by noelparisi** [comment]
@dylan-baskind the `attachment` error went away once I added `@types/react-tether` to the `package.json` though! 🎉

**Comment by dylan-baskind** [comment]
Heya @noelparisi - ah, of course :) Thanks for that.

And @danreeves happy to oblige, should just be a one liner, PR coming shortly.

**Comment by danreeves** [comment]
Thanks @dylan-baskind. Published in 2.0.2!
